### DISCUSSION TITLE
[本地部署]: 安装依赖之后用 python ChuanhuChatbot.py或者 python3 ChuanhuChatbot.py都不能启动

### DISCUSSION THREAD

**Discussion by 2661780070** [GaiZhenbiao/ChuanhuChatGPT#854] [Q&A]
### 是否已存在现有反馈与解答？

- [x] 我确认没有已有issue或discussion，且已阅读**常见问题**。

### 是否是一个代理配置相关的疑问？

- [X] 我确认这不是一个代理配置相关的疑问。

### 错误描述

本人小白，跟着教程一步步做的， 安装完依赖输入什么都不能启动，python版本是3.11，折腾了一下午，卡在最后一步了

### 复现操作

本人小白，跟着教程一步步做的， 安装完依赖输入什么都不能启动，python版本是3.11，折腾了一下午，卡在最后一步了

### 运行环境

```markdown
- OS: 
- Browser: 
- Gradio version: 
- Python version:
```

**Comment by Keldos-Li** [comment]
Hi，我能理解您烦躁的心情，但还是希望您能提供更多的信息，不然我们也没法帮到你。你是根据 [这个步骤](https://github.com/GaiZhenbiao/ChuanhuChatGPT/wiki/%E4%BD%BF%E7%94%A8%E6%95%99%E7%A8%8B#%E6%9C%AC%E5%9C%B0%E9%83%A8%E7%BD%B2) 本地安装部署的嘛？有没有您具体每步操作的截图？在启动时终端有何报错与输出？

**Comment by 2661780070** [comment]
是的，我电脑截屏坏了，直接复制过来是
Requirement already satisfied: mypy-extensions>=0.3.0 in c:\users\zhang'san\appd
ata\local\programs\python\python311\lib\site-packages (from typing-inspect<1,>=0
.4.0->dataclasses-json<0.6.0,>=0.5.7->langchain==0.0.173->-r requirements.txt (l
ine 10)) (1.0.0)
Requirement already satisfied: pycparser in c:\users\zhang'san\appdata\local\pro
grams\python\python311\lib\site-packages (from cffi>=1.12->cryptography>=36.0.0-
>pdfminer.six==20221105->pdfplumber->-r requirements.txt (line 13)) (2.21)

zhang'san@LAPTOP-VJ0GL0RM MINGW64 ~/Desktop/ChuanhuChatGPT-main
$ python ChuanhuChatbot.py

zhang'san@LAPTOP-VJ0GL0RM MINGW64 ~/Desktop/ChuanhuChatGPT-main
$ python3 ChuanhuChatbot.py

zhang'san@LAPTOP-VJ0GL0RM MINGW64 ~/Desktop/ChuanhuChatGPT-main
$
没有报错，也没有输出

**Comment by Keldos-Li** [comment]
可能要请您检查下是否正确配置与安装了Python，是否将python（python3）加入了环境变量。
试着在终端中执行
```
python --version
```
或
```
python3 --version
```
看看能否提示python版本？

**Comment by MZhao-ouo** [comment]
直接执行python呢

**Comment by 2661780070** [comment]
![Snipaste_2023-08-01_20-23-31](https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/141136108/342c700d-3467-4253-b054-09be6e761935)
![Snipaste_2023-08-01_20-20-11](https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/141136108/ed85b126-c60b-41e3-b719-41b54454241a)

**Comment by 2661780070** [comment]
我重新安装了python并且设置环境变量，但还是不行，麻烦大佬看一下吧

**Reply by Keldos-Li** [reply]
重启一下电脑？

**Reply by 2661780070** [reply]
感谢帮助，今天重启电脑，可以正常启动了

**Comment by ArnoWorkShop** [comment]
您好！不好意思我也遇到相同的問題，報錯如下：

C:\Users\AG09\Desktop\AI clone\ChuanhuChatGPT-main>python ChuanhuChatbot.py
Traceback (most recent call last):
  File "C:\Users\AG09\Desktop\AI clone\ChuanhuChatGPT-main\ChuanhuChatbot.py", line 214, in <module>
    keyTxt = gr.Textbox(show_label=True, placeholder=f"在这里输入你的OpenAI API-key...", value=initial_keytxt, label="API Key").style(container=True)
AttributeError: 'Textbox' object has no attribute 'style'

![image](https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/159093406/b347597f-e5e9-4aa4-b17e-5c4a684db854)
![image](https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/159093406/a34b58bc-7640-462b-af03-5684d8dde133)

我也都是看著教學視頻步驟一步一步做的
